engine/renderer/private/gpu_scene.cpp

```diff
@@ -115,6 +119,11 @@ void GPUScene::UpdateObjectInstancesData(uint32_t frameIndex)
         instances[count].materialIndex = mesh->material.Index();
         instances[count].boundingRadius = mesh->boundingRadius;
 
+        if (count == 8038)
```

Comment:
If we were to load a model before this one, this would no longer be correct, right?

Reply:
yes correct, this was just some a test condition for my breakpoint that i forgot to remove